spreadsheet_dashboard_oca/models/spreadsheet_dashboard.py

```diff
@@ -30,3 +32,13 @@ def _inverse_spreadsheet_raw(self):
             record.data = base64.encodebytes(
                 json.dumps(record.spreadsheet_raw).encode("UTF-8")
             )
+
+    def _compute_can_edit(self):
+        """We can edit if the record doesn't have XML-ID, or the XML-ID is noupdate=1"""
```

Comment:
If we export a record (compatible for import) an xml id is generated. 
With the current algorithm, if we export a record, it will become not editable. Don't you think ? 

Quite theoritical use case though.

Reply:
Not really, as the XML-ID entry `__export__.<whatever>` will be noupdate=1.

Reply:
> Not really, as the XML-ID entry __export__.<whatever> will be noupdate=1.

I just tested in a 16.0 CE database. 
- create a new product template, via UI

```
# select id, name, categ_id, type from product_template where id = 28;
 id |            name             | categ_id | type  
----+-----------------------------+----------+-------
 28 | {"en_US": "my new product"} |        1 | consu
```

- export the product via UI, (checking compatible for import)

```
select id, module, name, model, res_id, noupdate from ir_model_data where res_id = 28 and model = 'product.template';
  id   |   module   |             name             |      model       | res_id | noupdate 
-------+------------+------------------------------+------------------+--------+----------
 15659 | __export__ | product_template_28_c3c1d954 | product.template |     28 | f

```
If I understand correctly, noupdate = False, so can_edit = False

Reply:
Ouch, I didn't expect that one... Anyway, fixed :wink: